charts/vmclarity/values.yaml

```diff
@@ -396,6 +396,8 @@ gateway:
       drop:
         - ALL
 
+  service: {}
```

Comment:
In the case of the Kind cluster, we are using `extraPortMappings` for exposing the API to the localhost. So we need to add the possibility to choose NodePort instead of clusterIP.